Integrations/Elasticsearch_v2/Elasticsearch_v2.py

```diff
@@ -0,0 +1,257 @@
+    meta_headers = ['Query', 'took', 'timed_out', 'total', 'max_score', 'Server', 'Page', 'Size']
+    return search_context, meta_headers, hit_tables, hit_headers
+
+
+def get_total_results(response_dict):
+    total_results = response_dict.get('hits', {}).get('total')
+    if not str(total_results).isdigit():
+        # if in version 7 - total number of hits has value field
+        total_results = total_results.get('value')
+        total_dict = response_dict.get('hits').get('total')
+
+    else:
+        total_dict = {
+            'value': total_results,
+        }
+
+    return total_dict, total_results
+
+
+def search_command():
+    index = demisto.args().get('index')
+    query = demisto.args().get('query')
+    fields = demisto.args().get('fields')  # fields to display
+    explain = 'true' == demisto.args().get('explain')
+    base_page = int(demisto.args().get('page'))
+    size = int(demisto.args().get('size'))
+    sort_field = demisto.args().get('sort-field')
+    sort_order = demisto.args().get('sort-order')
+
+    es = Elasticsearch(hosts=[SERVER], connection_class=RequestsHttpConnection,
+                       http_auth=(USERNAME, PASSWORD), verify_certs=INSECURE)
+
+    que = QueryString(query=query)
+    search = Search(using=es, index=index).query(que)[base_page:base_page + size]
+    if explain:
+        # if 'explain parameter is set to 'true' - adds explanation section to search results
+        search = search.extra(explain=True)
+
+    if fields is not None:
+        fields = fields.split(',')
+        search = search.source(fields)
+
+    if sort_field is not None:
+        search = search.sort({sort_field: {'order': sort_order}})
+
+    response = search.execute().to_dict()
+    total_dict, total_results = get_total_results(response)
+    search_context, meta_headers, hit_tables, hit_headers = results_to_context(index, query, base_page,
+                                                                               size, total_dict, fields, response)
+    return_outputs(tableToMarkdown('Search Metadata:', search_context, meta_headers, removeNull=True) + '\n'
+                   + tableToMarkdown('Hits:', hit_tables, hit_headers, removeNull=True),
+                   {
+                        'Elasticsearch.Search(val.Query == obj.Query && val.Index == obj.Index'
+                        '&& val.Server == obj.Server && val.Page == obj.Page'
+                        '&& val.Size == obj.Size)': search_context
+                   },
+                   response)
+
+
+def test_func():
+    try:
+        res = requests.get(SERVER, auth=(USERNAME, PASSWORD), verify=INSECURE)
+        if res.status_code >= 400:
+            return_error("Failed to connect, Got the following error: " + HTTP_ERRORS[int(res.status_code)])
+
+    except requests.exceptions.RequestException:
+        return_error("Failed to connect, Check Server URL and Port number")
+
+    if demisto.params().get('isFetch'):
+        str_error = ''
```

Comment:
change to 
```suggestion
        str_error = []
```

then do